.github/workflows/rust-parallel-calls-example.yml

```diff
@@ -42,3 +42,26 @@ jobs:
           dfx start --background
           make test
           popd
+  rust-parallel-calls-pocket-ic-linux:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+        with:
+          submodules: recursive
+      - name: Provision Linux
+        run: bash .github/workflows/provision-linux.sh
+      - name: Install PocketIC server
+        uses: dfinity/pocketic@main
+      - name: Test parallel calls with PocketIC
+        run: |
+          pushd rust/parallel_calls
+          cargo build --release --target wasm32-unknown-unknown -p callee
+          cargo build --release --target wasm32-unknown-unknown -p caller
+          export CALLER_WASM=$(cargo build --target wasm32-unknown-unknown -p caller --message-format=json \
+            | jq -r 'select(.reason == "compiler-artifact") | .filenames[] | select(endswith(".wasm"))')
+          export CALLEE_WASM=$(cargo build --target wasm32-unknown-unknown -p callee --message-format=json \
+            | jq -r 'select(.reason == "compiler-artifact") | .filenames[] | select(endswith(".wasm"))')
+          cargo run
```

Comment:
using cargo test (and `#[test]` instead of a main function) is more common

Reply:
Agreed in principle but I wrote this test to be run from the command line (as specified in the `README`). Doing this as a test would make it a bit more complicated for the user (they'd have to pass flags to see the output, put the `pocket-ic` binary in the right directory etc). So I think it's a bit easier for them to keep it as a binary.

Reply:
> put the pocket-ic binary in the right directory etc

this is necessary anyway, but I get your point with "--nocapture"